thrust/testing/copy.cu

```diff
@@ -17,6 +17,12 @@
 
 #include <unittest/unittest.h>
 
+#if _CCCL_COMPILER(GCC, >=, 11)
```

Comment:
This seems unrelated -- just checking whether it's intended.

Reply:
I need `THRUST_DISABLE_BROKEN_GCC_VECTORIZER` as it triggers unrelated errors. Should I be changing the GCC value? Not sure which one to use. I took it from here https://github.com/NVIDIA/cccl/blob/98482bbc6060b05a69d34c2cf7da9330b20254da/thrust/testing/replace.cu#L9

Reply:
You'd have to check which versions it's failing on to be sure. But it's probably safe to just use 11 if CI is happy with it.